components/scream/src/physics/p3/tests/p3_rain_sed_unit_tests.cpp

```diff
@@ -52,20 +52,21 @@ static void run_bfb_rain_vel()
     {5.6298E-05, 1.1129E+00, 1.6576E+02},
     {1.0000E-02, 1.0774E+00, 1.9436E+02},
 
-    {0.0       , 1.3221E+00, 6.2964E+05},
+    {1.1030E-04, 1.3221E+00, 6.2964E+05},
```

Comment:
Maybe there was a reason to test 0.0? Like stress-testing some conditionals?

Reply:
@bartgol , yes, I'm sure that was the intention here. The "real" calls to compute_rain_fall_velocity occur within a block of code in rain_sedimentation that's already protected by qr_small checks. I did not want to add a check for the sake of unit testing that would be redundant and potentially reduce performance in the context of the full system.